### PR TITLE
 Add code tabs for _tour/case-classes

### PR DESCRIPTION
**Pull request by flomebul**

### CODE REVIEW

File: _tour/case-classes.md [hunk 1]
```diff
@@ -15,36 +15,64 @@ Case classes are like regular classes with a few key differences which we will g
...
+
+{% endtabs %}
+
 Notice how the keyword `new` was not used to instantiate the `Book` case class. This is because case classes have an `apply` method by default which takes care of object construction.
 
 When you create a case class with parameters, the parameters are public `val`s.
+
+{% tabs case-classe_Message_define %}
+
+{% tab 'Scala 2 and 3' for=case-Message_define %}
```
Comment (bishabosha) [inline _tour/case-classes.md:37]:
```suggestion
{% tab 'Scala 2 and 3' for=case-classe_Message_define %}
```

File: _tour/case-classes.md [hunk 2]
```diff
@@ -15,36 +15,64 @@ Case classes are like regular classes with a few key differences which we will g
...
+{% endtabs %}
+
 You can't reassign `message1.sender` because it is a `val` (i.e. immutable). It is possible to use `var`s in case classes but this is discouraged.
 
 ## Comparison
 Instances of case classes are compared by structure and not by reference:
+
+{% tabs case-classe_Message_compare %}
+
+{% tab 'Scala 2 and 3' for=case-Message_compare %}
```
Comment (bishabosha) [inline _tour/case-classes.md:56]:
```suggestion
{% tab 'Scala 2 and 3' for=case-classe_Message_compare %}
```

File: _tour/case-classes.md [hunk 3]
```diff
@@ -15,36 +15,64 @@ Case classes are like regular classes with a few key differences which we will g
...
+{% endtabs %}
+
 Even though `message2` and `message3` refer to different objects, the value of each object is equal.
 
 ## Copying
 You can create a (shallow) copy of an instance of a case class simply by using the `copy` method. You can optionally change the constructor arguments.
+
+{% tabs case-classe_Message_copy %}
+
+{% tab 'Scala 2 and 3' for=case-Message_copy %}
```
Comment (bishabosha) [inline _tour/case-classes.md:75]:
```suggestion
{% tab 'Scala 2 and 3' for=case-classe_Message_copy %}
```

File: _tour/case-classes.md (hunk 2 above)
Comment (flomebul) [reply _tour/case-classes.md:56]: Oups...

**Review by bishabosha — APPROVED** [review]

**Comment by bishabosha** [comment]
thanks!

**Pull request merged**